frontend/test/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file is part of the ALICE Electronic Logbook v2, also known as Jiskefet.
+ * Copyright (C) 2020  Stichting Hogeschool van Amsterdam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/* eslint-disable */
+
+const assert = require('assert')
+const puppeteer = require('puppeteer')
+const path = require('path');
+const { HttpServer } = require('@aliceo2/web-ui');
+
+const PORT = 3001
+
+describe('Frontend', () => {
+    before(() => {
+      http = new HttpServer({ port: PORT });
+      http.addStaticPath(path.resolve(__dirname, '../lib/public'));
+    })
+
+    before(async () => {
```

Comment:
I believe async calls require the usage of the `done` callback, could you verify that for the other async usages as well.

Reply:
@mvegter I don't fully understand you. The puppeteer is functional without the `done` callback.

Reply:
because you have tell mocha when you are done, try executing the test without the `--exit`.

Reply:
I've tried using the done callback but it gives me an error about that callback not being compatible with Promises.

`Resolution method is overspecified. Specify a callback *or* return a Promise; not both.`
I don't think it is needed.

Reply:
@MauritsioRK  What was the code that you used? Could you comment it below.